modules/local/mhcnuggets.nf

```diff
@@ -0,0 +1,29 @@
+process MHCNUGGETS {
+    label 'process_low'
+    tag "${metadata.sample}"
+
+    conda "bioconda::mhcnuggets=2.4.0"
```

Comment:
There is already version 2.4.1 existing in bioconda, so you could update here and the [container link](https://quay.io/repository/biocontainers/mhcnuggets?tab=tags)